deps/rabbit/src/rabbit.erl

```diff
@@ -467,6 +467,7 @@ stop() ->
                     Product = product_name(),
                     ?LOG_INFO("~ts is asked to stop...", [Product],
                               #{domain => ?RMQLOG_DOMAIN_PRELAUNCH}),
+                    rabbit_db:clear_init_finished(),
```

Comment:
`rabbit:stop/0` is not the real stop function, it's a helper only. Could you please move the call to the real `rabbit` stop function, `stop/1`?